.github/workflows/main.yml

```diff
@@ -49,6 +60,47 @@ jobs:
   required-checks-passed:
     name: All required checks passed
     runs-on: ubuntu-latest
-    needs: [documentation, lint-build-test]
+    needs: [documentation, build-lint-test]
```

Comment:
I think this will also need to be changed in repo settings after the merge.

Reply:
What would need to be changed?